libcontainer/factory_linux.go

```diff
@@ -35,6 +35,14 @@ var (
 // provided init binary path and arguments.
 func InitArgs(args ...string) func(*LinuxFactory) error {
 	return func(l *LinuxFactory) error {
+		if len(l.InitArgs) > 0 {
```

Comment:
We'll that will teach me to re-type instead of doing a partial cherry-pick won't it!